tests/integration/modal-auth.spec.holo.js

```diff
@@ -127,7 +131,47 @@ describe('Authentication Flow', () => {
     expect(callRegistry).toEqual({
       'chat.list_all_messages': 'done',
       'chat.refresh_chatter': 'done',
-      'profile.get_my_profile': 'done',
+      'profile.get_my_profile': 'done'
     })
   })
+
+  it('renders the sign-up page when provided sign-up uri search param', async () => {
+    await setupPage(page, callRegistry, `http://localhost:${serverPorts.ui}/dist/index.html?signup`, { waitForNavigation: true })
+    await wait(1000)
+
+    // verify page
+    const pageTitle = await page.title()
+    expect(pageTitle).toBe('Elemental Chat')
+
+    // *****************
+    // Arrive on Sign Up
+    // *****************
+    // wait for the modal to load
+    await page.waitForSelector('iframe')
+    const iframe = await findIframe(page, chaperoneUrlCheck.local)
+    const chaperoneModal = await iframe.evaluateHandle(() => document)
+
+    const [signUpTitle] = await findElementsByText('h2', 'Create Login Credentials', chaperoneModal)
+    expect(signUpTitle).toBeTruthy()
+  })
+
+  it('renders the sign-in page when provided sign-in uri search param', async () => {
+    await setupPage(page, callRegistry, `http://localhost:${serverPorts.ui}/dist/index.html?signin`, { waitForNavigation: true })
+    await wait(1000)
+
+    // verify page
+    const pageTitle = await page.title()
+    expect(pageTitle).toBe('Elemental Chat')
+
+    // *****************
+    // Arrive on Sign In
+    // *****************
+    // wait for the modal to load
+    await page.waitForSelector('iframe')
+    const iframe = await findIframe(page, chaperoneUrlCheck.local)
+    const chaperoneModal = await iframe.evaluateHandle(() => document)
+
+    const [signUpTitle] = await findElementsByText('h1', 'Elemental Chat Login', chaperoneModal)
```

Comment:
I don't think this is a great test. They will both be truthy all the time since they're only ever display:hidden, not removed from the DOM. Maybe you could call `click()` on it just to check that it is visible? I think that `click` fails when it's invisible